release_notes/versioning-policy.adoc

```diff
@@ -0,0 +1,51 @@
+[id='ocp-4-1-versioning-policy']
+= {product-title} Versioning Policy
+include::modules/common-attributes.adoc[]
+:context: release-notes
+
+toc::[]
+
+{product-title} provides strict backwards compatibility guarantees for all
+supported APIs, excluding alpha APIs (which may be changed without notice) and
+beta APIs (which may occasionally be changed in a non-backwards compatible
+manner).
+
+{product-title} 3.11 is compatible with {product-title} {product-version}. Red
```

Comment:
What do you mean by this first sentence? It makes me think that you can upgrade from 3.11 to 4.1, and that's not true.